### ISSUE TITLE
Microsoft Playwright Testing only works with Playwright test runner.

### ISSUE THREAD

**Issue by vvs11** [microsoft/playwright-testing-service#16]
Currently Microsoft Playwright Testing works only with Playwright test runner. Other test runners are not supported. 

AB#1903435

**Comment by PeterHevesi** [comment]
Yeah, I would be so happy if you create it also for Playwright.NET :)

**Comment by marlowp** [comment]
Does this mean that only tests written using Node.js can be used with the Microsoft Playwright Testing service?

**Comment by PeterHevesi** [comment]
@marlowp, yes, now it only support Node.js, so please upvote this issue, so they create support for other Playwright frameworks :) 
They should start with Playwright.NET :D

**Comment by AClerbois** [comment]
Upvote 👍

**Comment by vvs11** [comment]
We have added samples and instructions on how to configure Playwright Testing service for MSTest and NUnit test runner. Please try the steps and let us know the feedback. Here's the link to the samples: https://github.com/microsoft/playwright-testing-service/tree/main/samples/.NET

**Comment by RazvanFilipescu** [comment]
@vvs11  This is just the first version, right? The reporting is very basic, I cannot even see if the test has passed or failed in the Activity Log. Are there reporting features (maybe across test runs) in work?

**Comment by vvs11** [comment]
@RazvanFilipescu , yes this is the first version and right now, the .NET support is experimental. 
We have reporting on our roadmap but I'd love to understand your use-case and how you are thinking about cross test runs reporting feature. We have an open feature request for reporting here: https://github.com/microsoft/playwright-testing-service/issues/47 . Please upvote and share your use-case to help us build a better service.

**Comment by puagarwa** [comment]
Just putting a clarification around our support for other runners, Service does support connecting from any runner using playwright library via manual connect option https://github.com/microsoft/playwright-testing-service/issues/70
